### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by ghost** [breezy-weather/breezy-weather#1240] [Ideas]
*

**Comment by papjul** [comment]
No, Accrescent developers are asking users to not contact app developers until it is stable. Even when it will be stable, it might miss features we need.

If your request is to have a secure way instead, then you can compare SHA values of downloaded APK in the GitHub Releases section: https://github.com/breezy-weather/breezy-weather/releases

Note that once you installed the app, Android will check the signature before installing any update.
